frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -606,7 +606,7 @@ export const BillingProduct = ({ product }: { product: BillingProductV2Type }):
                     >
                         <div>
                             <h4 className={`${!upgradePlan ? 'text-success-dark' : 'text-warning-dark'}`}>
-                                You're on the {currentPlan.name} plan for {product.name}.
+                                You're on the {currentPlan?.name} plan for {product.name}.
```

Comment:
Should maybe have a fallback name otherwise it will read "You're on the plan for ..."

Reply:
This is only relevant for dev instances. On self-hosted they don't see the billing page, and on cloud they are defaulted into a plan. On dev we only see this because the instances aren't licensed. So maybe that's what we should say here...